src/workers/python/PythonWorker.worker.ts

```diff
@@ -65,16 +64,19 @@ class PythonWorker extends Backend {
     }
 
     override async _runCodeInternal(code: string): Promise<any> {
-        await this.pyodide.loadPackagesFromImports(code);
         if (this.initialized) {
             // run the code, potentially polluting the namespace
             // Functions and variables defined by the user become global
             // We need them to be global to let doctest work out of the box
```

Comment:
These can be removed :tada:

Reply:
Required a small change to ensure input was also correctly overriden, but this has been done now 🎉